src/System.DirectoryServices.AccountManagement/tests/ComputerPrincipalTest.cs

```diff
@@ -34,8 +35,13 @@ public void Ctor_NullSamAccountName_ThrowsArgumentException()
         [Fact]
         public void Ctor_EmptySamAccountName_ThrowsArgumentNullException()
         {
-            var context = new PrincipalContext(ContextType.Machine);
-            AssertExtensions.Throws<ArgumentNullException>("Principal.SamAccountName cannot be null or empty.", () => new ComputerPrincipal(context, string.Empty, "password", enabled: true));
+            RemoteInvoke(() =>
+            {
+                CultureInfo.CurrentUICulture = CultureInfo.InvariantCulture;
```

Comment:
For good hygiene, I would set CurrentUiCulture back to it’s original value after you are done.

Reply:
> I would set CurrentUiCulture back to it’s original value after you are done

It is executed in external process that will closed at end.